src/AppSearch/Request/GetTopClicksAnalytics.php

```diff
@@ -42,27 +45,27 @@ public function __construct(string $engineName)
 	 */
 	public function setQuery(string $query): self
 	{
-		$this->queryParams['query'] = $query;
+		$this->body['query'] = $query;
 		return $this;
 	}
 
 
 	/**
-	 * @param int $currentPage The page to fetch. Defaults to 1
+	 * @param int $query The page to fetch. Defaults to 1
 	 */
-	public function setCurrentPage(int $currentPage): self
+	public function setCurrentPage(int $query): self
```

Comment:
The name of the argument here has changed such that it is now inconsistent with the name of the function. Is this intentional? It seems potentially confusing.

Reply:
It looks like a bug in the code generator, I'll have a look. Thanks!

Reply:
I fixed the code generator and update the PR with the correct code.